.github/workflows/CI-tag-orchestrator.yml

```diff
@@ -5,54 +5,132 @@ on:
     tags:
       - "[0-9]+.[0-9]+.[0-9]+-[0-9]+.[0-9]+.[0-9]+*"
   workflow_dispatch:
+    inputs:
+      git_tag:
+        type: string
```

Comment:
What is the reason to take user input for git tag? :thinking: setup_env.sh scrip is still checking if TAG exist in the upstream and derives from PROD or DEV branches.

Reply:
Yes, to allow for a manual trigger if needed.

It was mostly useful for testing, but I can also see a scenario where you want to rerun it. If a test failed and it gets fixed, for example.